src/Lithnet.GoogleApps/GmailServiceRequestFactory.cs

```diff
@@ -109,9 +150,9 @@ public void RemoveSendAs(string id, string sendas)
         {
             id.ThrowIfNotEmailAddress();
 
-            GmailService service = this.GetService(id);
+            GmailService service = this.GetService(id, DefaultTimeout);
             UsersResource.SettingsResource.SendAsResource.DeleteRequest request = service.Users.Settings.SendAs.Delete(id, sendas);
-            request.ExecuteWithRetry(RetryEvents.BackoffOAuthNotFound, this.RetryCount, 5);
+            request.ExecuteWithRetry(RetryEvents.BackoffOAuth, this.RetryCount, 5);
```

Comment:
Any reason to remove the 'NotFound' policy?

Reply:
I think the instinct was to not retry the Delete request if the SendAs address does not exists (then we cannot delete it anyway). I may be missing something here though. Should I revert?